app/views/components/_autocomplete.html.erb

```diff
@@ -6,21 +6,20 @@
   type ||= nil
   size ||= nil
   hint ||= nil
+  width ||= nil
+  css_classes = %w(app-c-autocomplete govuk-form-group)
+  css_classes << "app-c-autocomplete--#{width}" if width
```

Comment:
could this just be called classes? I feel like the css prefix is redundant

Reply:
It's modifier class, which means that it should be marked as such using the `--narrow` suffix according to the adopted BEM convention. We could explicitly have `app-c-autocomplete--narrow` instead of a variable passed within.

Reply:
I'm not sure we're talking about the same thing. 

I was implying that `css_classes` as a variable name might want to change as they're just classes for the HTML element and we don't need to differentiate them from other classes.

Reply:
Ah, I got that wrong. We currently use a mix of `component_classes` and `css_classes` within our applications and a mix of `css_classes` and `[component_name]_classes` in govuk_publishing_components. Not very opinionated to be honest, but if we decide on one form I would update (at least the application level) for consistency.

Reply:
Sure, well if css_classes is already consistent then that's cool. I'd be voting we don't use that terminology though.